src/features/WordTable/index.tsx

```diff
@@ -48,7 +48,7 @@ const WordTable = ({
     ? [...cards].reverse() //don't mutate the original array or bad things will happen...
     : fuse.search(searchInput).map((result) => result.item);
   const activeCards = filteredCards.filter((card) =>
-    activeCardIds.includes(card.id)
+    activeCardIds.includes(card.id) && card.view === true
```

Comment:
`card.view` is a boolean already, so it could be changed to `&& card.view`. I would suggest moving this filter to the `App.tsx` file, so only relevant card entries get passed down to WordTable. (It'll make debugging easier in the future as well.)